### DISCUSSION TITLE
Compatibility with Nooelec NESDR v4?

### DISCUSSION THREAD

**Discussion by Jcat2b** [jopohl/urh#978] [Q&A]
Hello,

Do you know if the software is compatible with Nooelec NESDR v4?
If not, should we make a record with another software? Then open it with URF for the analyze right?

Thank you.

**Comment by jhit** [comment]
I have this device it works OK. But it consumes a lot of energy and actually led to hardware failure of my raspberry pi 4b. 
I would recommend to only using it with a powered USB hub to eliminate the risk that you harm your computer.
